lib/ansible/modules/network/fortimanager/fmgr_secprof_web.py

```diff
@@ -28,9 +28,9 @@
 module: fmgr_secprof_web
 version_added: "2.8"
 author:
-    - Luke Weighall (@lweighall)
-    - Andrew Welsh (@Ghilli3)
-    - Jim Huber (@p4r4n0y1ng)
+    - Luke Weighall (lweighall)
```

Comment:
@ftntcorecse was there any specific reason for removing the `@` as ansible modules follow author with `@` sign.